services/integration.sh

```diff
@@ -88,21 +97,26 @@ function run() {
         echo -e "\n\nWARNING: log output is buffered and may not show on your screen!\n\n"
         UNBUFFERED=''
     fi
-    ( ( cd "${DIR}/${service}" && "${TOP_LEVEL}/dist/${service}" -c "${service}${instance}.integration${integration_file_extension}" ) || kill_all) \
+    ( ( cd "${DIR}/${service}" && "${TOP_LEVEL}/dist/${service}" -c "${configfile}" ) || kill_all) \
         | sed ${UNBUFFERED} -e "s/^/$(tput setaf ${colour})[${service}] /" -e "s/$/$(tput sgr0)/" &
 }
 
 
-check_prerequisites
-
-run brig "" ${green}
-run galley "" ${yellow}
-run gundeck "" ${blue}
-run cannon "" ${orange}
-run cannon "2" ${orange}
-run cargohold "" ${purpleish}
-run spar "" ${orange}
-run federator "" ${blue}
+if [[ $INTEGRATION_CARGOHOLD_ONLY_COMPAT -eq 1 ]]; then
+    source "${CARGOHOLD_COMPAT_CONFIG_FOLDER}/env.sh"
```

Comment:
We could also do relevant parts of `check_prerequisites` here, but not a big deal now that this is not run routinely.